docs/source/virtual_rainforest/module_overview.md

```diff
@@ -100,52 +100,70 @@ Further theoretical background for the soil module can be found [here](./soil/so
 
 The abiotic module provides the microclimate and hydrology for the Virtual Rainforest.
 Using a small set of input variables from external sources such as WFDE5
-({cite}:`WFDE5-2020`) or regional climate models, the module calculates atmospheric and
+({cite}`WFDE5-2020`) or regional climate models, the module calculates atmospheric and
 soil parameters that drive the dynamics of plants, animals, and microbes at different
-vertical levels. Two subroutines, the energy balance and the water balance, provide the
-following variables at required levels:
+vertical levels. Four subroutines - the radiation balance, the energy balance, the water
+balance, and the atmospheric $\ce{CO_{2}}$ balance - provide the following variables at
+different vertical levels:
 
-- Net radiation ($R_N$) and Photosynthetic photon flux density ($PPFD$)
-- Air Temperature ($T\_{air}$)
-- Relative humidity ($RH$) and vapor pressure deficit ($VPD$)
-- Soil Temperature ($T\_{soil}$)
-- Soil moisture ($W\_{soil}$)
-- Runoff ($RO$), mean vertical flow ($VF$) and drainage ($D$)
+- Net radiation and Photosynthetic photon flux density
+- Air temperature, relative humidity, and vapor pressure deficit
+- Soil temperature and soil moisture
+- Atmospheric $\ce{CO_{2}}$ concentration
+- above- and belowground runoff, mean vertical flow, and streamflow (at catchment scale)
 
 ### Vertical structure of atmosphere and soil
 
-The atmosphere is divided in four vertical layers:
+The atmosphere is divided in four vertical levels:
 
 1. the top of the canopy which links the external driver to the module,
-1. the upper canopy where most photosynthetic activity occurs,
-1. the understorey where most large animal are active, and
-1. the near surface which homes ground-dwelling organisms and links the atmosphere to
-   the top soil layer.
+2. the upper canopy where most photosynthetic activity occurs (~20 m to  top of canopy,
```

Comment:
BTW you can get away with using `1. 1. 1. 1.` for a numbered list, I think it's actually the preferred style as it's numbered automatically rather than relying on always getting the order of the numbers in the list right. Minor thing though and no need to change